lib/msal-browser/package.json

```diff
@@ -10,7 +10,7 @@
         "type": "git",
         "url": "https://github.com/AzureAD/microsoft-authentication-library-for-js.git"
     },
-    "version": "2.0.0-beta.0",
+    "version": "2.0.0-beta.1",
```

Comment:
The lockfile also has a version that needs to be bumped.

Reply:
done.